src/overlays/actors/ovl_En_Knight/z_en_knight.c

```diff
@@ -205,237 +380,4044 @@ ActorInit En_Knight_InitVars = {
     /**/ EnKnight_Draw,
 };
 
-#endif
+u16 sKnightWalkSfx[3] = { NA_SE_EN_BOSU_WALK, NA_SE_EN_YASE_WALK, NA_SE_EN_DEBU_WALK };
+u16 sKnightPauseSfx[3] = { NA_SE_EN_BOSU_LAUGH_K, NA_SE_EN_YASE_PAUSE_K, NA_SE_EN_DEBU_PAUSE_K };
+u16 sKnightLaughSfx[3] = { NA_SE_EN_BOSU_LAUGH_K, NA_SE_EN_YASE_LAUGH_K, NA_SE_EN_DEBU_LAUGH_K };
+u16 sKnightAttackSfx[3] = { NA_SE_EN_BOSU_ATTACK_W, NA_SE_EN_YASE_ATTACK_W, NA_SE_EN_DEBU_ATTACK_W };
+u16 sKnightDamagedSfx[3] = { NA_SE_EN_BOSU_DAMAGE, NA_SE_EN_YASE_DAMAGE, NA_SE_EN_DEBU_DAMAGE };
+u16 sKnightDefeatedSfx[3] = { NA_SE_EN_BOSU_DEAD, NA_SE_EN_YASE_DEAD, NA_SE_EN_DEBU_DEAD };
+u16 sKnightVoiceSfx[3] = { NA_SE_EN_BOSU_DEAD_VOICE, NA_SE_EN_YASE_DEAD_VOICE, NA_SE_EN_DEBU_DEAD_VOICE };
+
+EnKnight* sIgosInstance; // Bosu (igos)
+EnKnight* sYaseInstance; // Yase (tall minion)
+EnKnight* sDebuInstance; // Debu (wide minion)
```

Comment:
I did think about this when I originally named these variables. I considered naming them based on rough English equivalents but couldn't settle on anything in particular so I just left it in Japanese. If you'd like to suggest some specific names I don't mind changing it.

Reply:
In terms of their appearance, I think in MM64, the only thing that changes is their head, right? I didn't see any limb-scaling code in here, and they use the same body DLs. For MM3D, though, you can see they *did* actually give them different body shapes; you can see that the bones for one of them are narrower than the bones of the other one:
![image](https://github.com/user-attachments/assets/e8692c6c-7b4b-4f37-8740-51c3cce20fee)
![image](https://github.com/user-attachments/assets/0047fd3a-e21a-4d0d-80d4-7dc31eeb5252)

I should also note that these guys have a couple of different "names" as a collective based on who you ask. The Zelda Encyclopedia calls them "lackeys":
![image](https://github.com/user-attachments/assets/ca711b3b-4d52-4f11-8471-df67d116a791)

Grezzo called them "aides", but they put them in a folder called "zako", which means something like "small fish; small fry". Someone more familiar with Japanese told me that "zako" is a common term for "scrub enemies":
![image](https://github.com/user-attachments/assets/c7fea848-70d0-42d1-aa70-9227bfb0513f)

So maybe something like `sThinLackeyInstance` and `sWideLackeyInstance`? Though instead of "lackey," I'd also be fine with "aide", "knight", "minion", etc. It's a bit tricky because they vary by so little in MM64, but I think both the SFX names and the way they changed them in MM3D make it pretty clear that we're supposed to distinguish them by their width. What do you think?

Reply:
Makes sense to me. Went with `sThinKnightInstance` and `sWideKnightInstance`